File: doc/codes/python/examples/api/compute/pattern/null.py
from autd3.geometry import Autd3, Geometry
from autd3_pattern import null

geometry = Geometry([Autd3([0.0, 0.0, 0.0], [1.0, 0.0, 0.0, 0.0])])

out = geometry.pattern_buffer()

# ANCHOR: api
null(out)
# ANCHOR_END: api

File: doc/codes/csharp/examples/api/compute/pattern/null.cs
using System.Numerics;
using AUTD3;

namespace DocSamples.ApiComputePatternNull;

internal static class Sample
{
    internal static void Run()
    {
        var geometry = new Geometry(new[] { new Autd3(Vector3.Zero) });

        var dst = geometry.PatternBuffer();
        // ANCHOR: api
        Pattern.Null(dst);
        // ANCHOR_END: api
    }
}
